application/models/Note.php

```diff
@@ -31,23 +30,37 @@ function add() {
 
 	function edit() {
 		$data = array(
-			'cat' => xss_clean($this->input->post('category')),
-			'title' => xss_clean($this->input->post('title')),
-			'note' => xss_clean($this->input->post('content'))
+			'cat' => $this->input->post('category', TRUE),
```

Comment:
why? it's the same - just a different notation ;)

Reply:
consistency